### PR TITLE
Update learnmsa to 1.2.0

### PR DESCRIPTION
**Pull request by felbecker**
Fixing failed autobump #41427

### CODE REVIEW

**Comment by felbecker** [comment]
@BiocondaBot please fetch artifacts

**Comment by felbecker** [comment]
@BiocondaBot please add label

File: recipes/learnmsa/meta.yaml [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
   noarch: python
-
+  
 requirements:
   host:
     - pip
     - python >=3.7
   run:
+    - libtiff
```
Comment (bgruening) [inline recipes/learnmsa/meta.yaml:24]: Why as run dependency? Is this missing in a downstream dependency?

Comment (felbecker) [reply recipes/learnmsa/meta.yaml:24]: Yes, but I don't know why it's suddenly missing. I do not directly depend on it.

**Comment by bgruening** [comment]
I hope it's ok if I close, the other PR is now in.

**Pull request closed without merging**